cowprotocol/accounting/slippage/slippage_per_transaction_4070059.sql

```diff
@@ -0,0 +1,33 @@
+-- This query return slippage per transaction
+--
+-- Parameters:
+--  {{start_time}} - the timestamp for which the analysis should start (inclusively)
+--  {{end_time}} - the timestamp for which the analysis should end (exclusively)
+--  {{blockchain}} - network to run the analysis on
+--
+-- The columns of the result are
+-- - block_time: time of settlement transaction
+-- - tx_hash: settlement transaction hash
+-- - solver_address: address of the solver executing the settlement
+-- - slippage_usd: USD value of slippage
+-- - slippage_native: value of slippage in native token
+--
+-- Results of the query are filtered to not include batches from excluded_batches.
+-- Batches are also excluded if there is a non-zero imbalance and no value (in native atomes).
+
+with excluded_batches as (
```

Comment:
I think this should be moved to the `slippage_per_solver` query.

Reply:
One goal was to have a query per transaction which is exactly what we use for total slippage to base an unusual slippage query on.

If you think that is not a requirement and having a discrepancy there if fine, I can move excluded batches somewhere else.

Maybe the better distinction is to merge the three queries slippage breakdown, slippage per transaction, slippage per solver into
- raw slippage (including breakdown and aggregation per transaction)
- slippage (with filtering, per transaction and per solver)

The first query would then be for debugging and could also leave out solver names, the second would be compatible with actual payment.